packages/react-core/src/layouts/Level/Level.d.ts

```diff
@@ -0,0 +1,16 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+import { Omit, OneOf } from '../../typeUtils';
+
+export const GutterSize: {
+  sm: 'sm';
```

Comment:
Let's use the, now merged, shared gutter size from styles/gutters.

Reply:
Yeah, sure, I forgot to update this, thanks for heads up.